tests/phpunit/tests/editor/disableBlockEditorForNavigationPostType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @group editor
+ *
+ * @covers ::_disable_block_editor_for_navigation_post_type
+ */
+class Tests_Editor_DisableBlockEditorForNavigationPostType extends WP_UnitTestCase {
+
+	const NAVIGATION_POST_TYPE     = 'wp_navigation';
+	const NON_NAVIGATION_POST_TYPE = 'wp_non_navigation';
+
+	/**
+	 * @ticket 56266
+	 *
+	 * @dataProvider data_test_should_only_disable_block_editor_for_navigation_post_types
```

Comment:
```suggestion
	 * @dataProvider data_should_only_disable_block_editor_for_navigation_post_types
```

Reply:
Fixed in d6ee08b6441d289e4009cbf66b298f3959fa54ad